extensions/resource-deployment/src/interfaces.ts

```diff
@@ -153,18 +153,33 @@ export interface DialogInfoBase {
 export interface DialogTabInfo extends PageInfoBase {
 }
 
-export interface PageInfoBase extends SharedFieldAttributes {
+export interface PageInfoBase extends FieldInfoBase {
 	title: string;
 	isSummaryPage?: boolean;
+	isSummaryPageAutoGenerated?: boolean;
```

Comment:
Why is this a property on every page? Shouldn't this just be at the wizard config level?

Reply:
Will move.

Reply:
fixed